circle.yml

```diff
@@ -0,0 +1,16 @@
+machine:
+  node:
+    version: 4
+
+dependencies:
+  pre:
+    - npm -g install codecov
```

Comment:
oh, it's just a pure CI dependency/I don't give a shit about the version/less annoying to use it globally